packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/User.kt

```diff
@@ -75,4 +81,7 @@ class User {
             inverseJoinColumns = arrayOf(JoinColumn(name = "dashboard_id", referencedColumnName = "id"))
     )
     var favoriteDashboards: Collection<Dashboard> = arrayListOf()
-}
\ No newline at end of file
+
+    @get:OneToMany(targetEntity = Role::class)
```

Comment:
Is there an annotation for just a `toMany` relationship since the role doesn't actually have a relationship to the users?

Reply:
Looking into it, this is functionally a `toMany` annotation since `roles` doesn't have a `ManyToOne` annotation on user so you can ignore this comment.

Reply:
There's one but I haven't found a code example and I get errors like `Could not determine type for: java.util.Collection, at table: navi_users, for columns: [org.hibernate.mapping.Column(roles)]`